pom.xml

```diff
@@ -269,7 +269,7 @@
     <version.org.springframework>3.2.2.RELEASE</version.org.springframework>
     <version.org.wildfly.arquillian>5.0.0.Alpha5</version.org.wildfly.arquillian>
     <version.postgresql>42.3.1</version.postgresql>
-    <version.quarkus>3.0.0.Alpha4</version.quarkus>
+    <version.quarkus>3.0.0.Alpha3</version.quarkus>
```

Comment:
Could you explain why we need to downgrade that dependency?

Reply:
@marcosgopen  Alpha4 is not compatible dependency which is throwing an error. It was just defined and not actively used in narayana. I checked with Alpha5 and it is working fine with it.